### PR TITLE
sysbuild: Add support for OpenThread

### PR DESCRIPTION
**Pull request by maciejbaczmanski**

### CODE REVIEW

File: doc/nrf/protocols/thread/certification.rst [hunk 1]
```diff
@@ -66,26 +66,26 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   Use the :ref:`ot_cli_sample` sample as a base, and apply the ``ci`` and ``multiprotocol`` snippets.
+   Use the :ref:`ot_cli_sample` sample as a base, apply the ``ci`` and ``multiprotocol`` snippets and if building for ``nrf5340dk_nrf5340`` target, set :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` flag.
```
Comment (nordicjm) [inline doc/nrf/protocols/thread/certification.rst]: this is a hwmv1 name

File: samples/openthread/cli/README.rst [hunk 2]
```diff
@@ -101,6 +103,8 @@ The following snippets are available:
 * ``ci`` - Disables boot banner and shell prompt.
 * ``multiprotocol`` - Enables Bluetooth LE support in this sample.
   Not compatible with the ``tcat`` snippet.
+  .. note::
+    When building with ``multiprotocol`` snippet, for the ``nrf5340dk_nrf5340`` build target, additional :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` option has to be set.
```
Comment (nordicjm) [inline samples/openthread/cli/README.rst]: hwmv1 name

File: samples/openthread/coap_client/README.rst [hunk 3]
```diff
@@ -125,6 +127,8 @@ The following snippets are available:
 * ``logging`` - Enables logging using RTT.
   For additional options, refer to :ref:`RTT logging <ug_logging_backends_rtt>`.
 * ``multiprotocol_ble`` - Enables the Multiprotocol Bluetooth LE extension.
+  .. note::
+    When building with ``multiprotocol_ble`` snippet, for the ``nrf5340dk_nrf5340`` build target, additional :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` option has to be set.
```
Comment (nordicjm) [inline samples/openthread/coap_client/README.rst]: etc.

File: doc/nrf/protocols/thread/certification.rst (hunk 1 above)
Comment (maciejbaczmanski) [reply doc/nrf/protocols/thread/certification.rst]: fixed

File: doc/nrf/protocols/thread/certification.rst [hunk 4]
```diff
@@ -66,26 +66,26 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   Use the :ref:`ot_cli_sample` sample as a base, and apply the ``ci`` and ``multiprotocol`` snippets.
+   Use the :ref:`ot_cli_sample` sample as a base, apply the ``ci`` and ``multiprotocol`` snippets and if building for ``nrf5340dk/nrf5340/cpuapp`` target, set :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` flag.
```
Comment (wiba-nordic) [inline doc/nrf/protocols/thread/certification.rst:69]:
```suggestion
   Use the :ref:`ot_cli_sample` sample as a base and apply the ``ci`` and ``multiprotocol`` snippets.
   If you are building for the ``nrf5340dk/nrf5340/cpuapp`` target, also set the :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` Kconfig option to ``y``.
```

File: doc/nrf/protocols/thread/certification.rst [hunk 5]
```diff
@@ -66,26 +66,26 @@ Complete the following steps to prepare for the certification tests:
...
      .. code-block::
 
-        -DSNIPPET="ci;multiprotocol"
+        -Dcli_SNIPPET="ci;multiprotocol"
         -DCONFIG_OPENTHREAD_LIBRARY=y
 
    .. note::
       The configuration option selects the precompiled OpenThread libraries.
-      The ``multiprotocol`` snippet enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
+      The ``multiprotocol`` snippet and :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` flag enable :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
```
Comment (wiba-nordic) [inline doc/nrf/protocols/thread/certification.rst:88]:
```suggestion
      The ``multiprotocol`` snippet and the :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` Kconfig option enable :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
```

File: samples/openthread/cli/README.rst [hunk 6]
```diff
@@ -101,6 +103,8 @@ The following snippets are available:
 * ``ci`` - Disables boot banner and shell prompt.
 * ``multiprotocol`` - Enables Bluetooth LE support in this sample.
   Not compatible with the ``tcat`` snippet.
+  .. note::
+    When building with ``multiprotocol`` snippet, for the ``nrf5340dk/nrf5340/cpuapp`` build target, additional :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` option has to be set.
```
Comment (wiba-nordic) [inline samples/openthread/cli/README.rst:107]:
```suggestion

  .. note::
    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` Kconfig option has to be set to ``y``.

```
Note the empty lines before and after the note.

File: samples/openthread/coap_client/README.rst [hunk 7]
```diff
@@ -125,6 +127,8 @@ The following snippets are available:
 * ``logging`` - Enables logging using RTT.
   For additional options, refer to :ref:`RTT logging <ug_logging_backends_rtt>`.
 * ``multiprotocol_ble`` - Enables the Multiprotocol Bluetooth LE extension.
+  .. note::
+    When building with ``multiprotocol_ble`` snippet, for the ``nrf5340dk/nrf5340/cpuapp`` build target, additional :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` option has to be set.
```
Comment (wiba-nordic) [inline samples/openthread/coap_client/README.rst:131]:
```suggestion

  .. note::
    When building with the ``multiprotocol_ble`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` Kconfig option has to be set to ``y``.
```

**Comment by maciejbaczmanski** [comment]
already added to PR: https://github.com/nrfconnect/sdk-nrf/pull/13153

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Pull request closed without merging**